### DISCUSSION TITLE
SEO suggestions

### DISCUSSION THREAD

**Discussion by khannzxc** [razonyang/hugo-theme-bootstrap#144] [Q&A] [answered]
As duplicate page titles and Meta descriptions are not welcome by google.
Is it a better idea to add just ```no index``` not ```no follow```  to the robots.txt file for categories and tags URLs such as
```
https://hbs.razonyang.com/en/tags/math/
```
So the tags and categories will serve the site purpose but will not index in the google search.

**Comment by khannzxc** [comment]
for SEO audit
[https://app.neilpatel.com/en/seo_analyzer](url)

**Comment by razonyang** [comment]
I have no much experience about SEO. It'll be a big change, I'd like to get more feedback and opinions before making a decision.

**Reply by khannzxc** [reply]
> I have no much experience about SEO. It'll be a big change, I'd like to get more feedback and opinions before making a decision.

yeah, I have seen a lot of Hugo themes all have the same structure about tags/categories.
so if we use the same tag and category name it will have the same title tags.

[https://hugo-theme-bootstrap.netlify.app/en/posts/shortcodes/]()
[https://hugo-theme-bootstrap.netlify.app/encategories/shortcode/]()

I once used not the best approach to minimize some duplications
```
{{ if .Data.Plural }} <meta name="description" content="{{$.Param
"description2"}}"> {{else}} <meta name="description" content="{{ $.Param
"description" }}" /> {{ end }}
```

Anyhow, we will focus on other important problems.

**Comment by razonyang** [comment, marked as answer]
But in reality, I don't think there should be the same tag and category. 
Even so, we can avoid the same title by adding types, for example `Foo - Tags - Site Title` and `Bar - Categories - Site Title`.

**Reply by razonyang** [reply]
@khannzxc FYI, fixed by 32a6104.
